features/polyExplorer/src/components/clusterStories/receivingCompanies.jsx

```diff
@@ -116,20 +116,32 @@ function Industries({ entities }) {
 }
 
 export default function ReceivingCompanies({ entities }) {
+    const [selectedRecievingCompaniesTab, setSelectedRecievingCompaniesTab] =
+        useState("companies-bar-chart-info");
+    const switchCompany = (tabId) => {
+        switch (tabId) {
+            case "companies":
+                setSelectedRecievingCompaniesTab("companies-bar-chart-info");
+                break;
+            case "industries":
+                setSelectedRecievingCompaniesTab(
+                    "industries-packed-circle-info"
+                );
+                break;
+            default:
```

Comment:
If this is the same as the case "companies", you can just skip case, and once you have only two options, possibly just use an `if`

Reply:
thought of using the switch as to keep things in similar convention throughout, for when there is more tabs in a different implementation. but yes, an if statement would be functionally equivalent and probably easier to read